### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by matildasvensson85**
Link to my deployed project: https://reverent-murdock-6eb4c8.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/CompletedTasks.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import { tasks } from '../reducers/tasks'
+import styled from 'styled-components'
+import { useSelector } from 'react-redux'
+import uniqid from 'uniqid'
+
+const CompletedSection = styled.section`
+    margin: 30px 0 0 0;
+    display: flex;
+    flex-direction: column;
+`
```
Comment (leelalaim) [inline code/src/components/CompletedTasks.js]: Nice that you have used styled components!

File: code/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    const amountOfTasks = useSelector((store) => store.tasks.taskList.length)
+
+    return (
+        <HeaderSection>
+            <LeftWrapper>
+                <AppTitle>Todo</AppTitle>
+           </LeftWrapper>
+           <RightWrapper>
+                {/* <DateTitle>Date</DateTitle> */}
```
Comment (leelalaim) [inline code/src/components/Header.js]: Remove the comments from the code :)

File: code/src/reducers/tasks.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import uniqid from 'uniqid'
+
+const taskList = [
+    { id: 1, taskText: 'Get milk', isDone: false },
+    { id: 2, taskText: 'Clean', isDone: false  },
+    { id: 3, taskText: 'Go running', isDone: false  }
+]
+
+const completedTasks = [
+    // { id: 4, taskText: 'Homework', isDone: true },
```
Comment (leelalaim) [inline code/src/reducers/tasks.js]: Remove the comments from the code :)

File: code/src/reducers/tasks.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+        deleteTask: (store, action) => {
+            const leftTasks = store.taskList.filter(task => task.id !== action.payload) 
+            store.taskList = leftTasks
+        },
+
+        
+        addTask: (store, action) => {
+            // store.taskList = [...store.taskList, action.payload]
+            // store.taskList.push(action.payload)
```
Comment (leelalaim) [inline code/src/reducers/tasks.js]: Remove the comments :)

**Review by leelalaim — APPROVED** [review]
Nice and neat code. Remove the comments that are not needed from the code. The live preview link was not working, check netlify.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
please check your link to the netlify because it returns "page not found"

**Review by KarolCislowski — APPROVED** [review]
Good job 👍🏻

**Pull request closed without merging**